force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js

```diff
@@ -496,6 +509,12 @@ export default class ServiceDeliveryRow extends LightningElement {
         });
     }
 
+    resetQuantityLabel() {
```

Comment:
We need to handle the case that a user picks a service with a unit of measure, but then changes to a service with no UoM.  In that case, the field label should change back to quantity.  Currently, it says with the last value.

Reply:
ah thank you, that was the scenario i couldn't remember. on it.